README.md

```diff
@@ -1 +1,22 @@
 # redshifter
+
+`redshifter` is responsible for getting data from any source into AWS Redshift for data analysis.
+
+## Motivation
+
+[AWS Redshift](http://aws.amazon.com/redshift/) is a columnar storage based data warehouse solution.
+It is optimized for analysis and business intelligence and has integration with many tools.
+
+## Features
+
+Right now this repository contains a single script that exports OAuth authorize events data from
+Mixpanel into Redshift via S3.
```

Comment:
:+1: seems easy to remove references to oauth here and in the filename `mixpanel_oauth_to_redshift.go`